### PR TITLE
Added new Class by Chandana

### PR DESCRIPTION
**Pull request by chandu4321**
mujeeb is very good friends of adhi

### CODE REVIEW

File: TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package testCasesOfTricentisAutomationModule;
+
+import org.testng.annotations.Test;
+
```
Comment (Bhoom12) [inline TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java]: add author

File: TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package testCasesOfTricentisAutomationModule;
+
```
Comment (Bhoom12) [inline TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java:2]: fgh

**Comment by Bhoom12** [comment]
do properly

File: TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java (hunk 1 above)
Comment (chandu4321) [reply TricentisInsurance/src/testCasesOfTricentisAutomationModule/Trial_001.java]: done

**Pull request merged**